js/views/onboardingModal.js

```diff
@@ -351,6 +351,10 @@ module.exports = baseModal.extend({
                 console.log(errorThrown);
               }
             });
+          } else {
```

Comment:
I don't know why the settings fail but the socket works either. Saving to profile worked normally, only settings didn't work.

It's ready for merge.